pkg/server/serverpb/status.proto

```diff
@@ -1376,13 +1376,18 @@ message HotRangesRequest {
   // If left empty, the request is forwarded to every node
   // in the cluster.
   // API: PUBLIC ALPHA
-  string node_id = 1 [(gogoproto.customname) = "NodeID"];
```

Comment:
chatted about this offline, we may want to consider keeping this field and using it to additionally populate the `nodes` field to avoid backwards compatibility concerns